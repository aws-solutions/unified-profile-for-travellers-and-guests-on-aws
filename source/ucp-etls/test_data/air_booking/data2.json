{
  "objectVersion": 10,
  "modelVersion": "1.0",
  "id": "TLIUNY",
  "externalIDs": null,
  "lastUpdatedOn": "2021-10-07T15:47:19.902359909Z",
  "createdOn": "2021-09-07T15:47:19.902359909Z",
  "lastUpdatedBy": "Victor Jakubowski",
  "createdBy": "Wayne Konopelski",
  "creationChannelId": "mobile",
  "price": {
    "status": null,
    "grandTotal": 0.0
  },
  "itinerary": {
    "from": "PEK",
    "to": "DEN",
    "departureDate": "2022-02-24",
    "departureTime": "02:48",
    "arrivalDate": "2022-02-24",
    "arrivalTime": "12:46",
    "duration": "9h58m0s",
    "segments": [
      {
        "rank": 0,
        "from": "PEK",
        "to": "DEN",
        "departureDate": "2022-02-24",
        "departureTime": "02:48",
        "arrivalDate": "2022-02-24",
        "arrivalTime": "12:46",
        "flightNumber": "RK0414",
        "inventory": {
          "totalSeats": 0,
          "inventoryByFareClass": null
        },
        "status": ""
      }
    ],
    "inventory": {
      "totalSeats": 0,
      "inventoryByFareClass": null
    },
    "status": "confirmed"
  },
  "return": {
    "from": "",
    "to": "",
    "departureDate": "",
    "departureTime": "",
    "arrivalDate": "",
    "arrivalTime": "",
    "duration": "",
    "segments": null,
    "inventory": {
      "totalSeats": 0,
      "inventoryByFareClass": null
    },
    "status": ""
  },
  "passengerInfo": {
    "passengers": [
      {
        "modelVersion": "1.0",
        "id": "1150274836",
        "lastUpdatedOn": "2021-10-24T06:55:23.532257551Z",
        "createdOn": "2021-07-24T06:55:23.532257551Z",
        "lastUpdatedBy": "June Rutherford",
        "createdBy": "Lina King",
        "emails": [
          {
            "type": "business",
            "address": "dangelowehner@stiedemann.info",
            "primary": false
          }
        ],
        "phones": [
          {
            "type": "",
            "number": "712.877.8663",
            "primary": true,
            "countryCode": 4
          },
          {
            "type": "personal",
            "number": "6672245434",
            "primary": true,
            "countryCode": 29
          }
        ],
        "addresses": [],
        "honorific": "Lady",
        "firstName": "Krista",
        "middleName": "Tony",
        "lastName": "Roob",
        "gender": "female",
        "pronoun": "she",
        "dateOfBirth": "2022-05-26",
        "language": {
          "code": "ee",
          "name": ""
        },
        "nationality": {
          "code": "GS",
          "name": ""
        },
        "jobTitle": "Supervisor",
        "parentCompany": "CareSet Systems",
        "loyaltyPrograms": [],
        "identityProofs": null
      },
      {
        "modelVersion": "1.0",
        "id": "6671697500",
        "lastUpdatedOn": "2023-01-13T17:48:56.394398113Z",
        "createdOn": "2022-12-13T17:48:56.394398113Z",
        "lastUpdatedBy": "Sanford Runte",
        "createdBy": "Anjali Kreiger",
        "emails": [],
        "phones": [],
        "addresses": [],
        "honorific": "Mx",
        "firstName": "Tanya",
        "middleName": "Constantin",
        "lastName": "Schmidt",
        "gender": "other",
        "pronoun": "they",
        "dateOfBirth": "1986-05-10",
        "language": {
          "code": "lv",
          "name": ""
        },
        "nationality": {
          "code": "PS",
          "name": ""
        },
        "jobTitle": "Architect",
        "parentCompany": "Wheaton World Wide Moving",
        "loyaltyPrograms": [
          {
            "id": "8922230558",
            "programName": "reward",
            "miles": 384375.0,
            "milesToNextLevel": 4057.0,
            "level": "platinium",
            "joined": "2022-12-13T17:48:56.394398113Z"
          }
        ],
        "identityProofs": null
      },
      {
        "modelVersion": "1.0",
        "id": "6759135163",
        "lastUpdatedOn": "2022-04-05T04:06:43.490493818Z",
        "createdOn": "2021-12-05T04:06:43.490493818Z",
        "lastUpdatedBy": "Brandt Corkery",
        "createdBy": "Lupe Boehm",
        "emails": [
          {
            "type": "personal",
            "address": "ricokemmer@powlowski.name",
            "primary": false
          },
          {
            "type": "personal",
            "address": "elisabethrunolfsson@kihn.io",
            "primary": false
          },
          {
            "type": "business",
            "address": "aliyahklein@erdman.name",
            "primary": false
          }
        ],
        "phones": [],
        "addresses": [],
        "honorific": "Ms",
        "firstName": "Cielo",
        "middleName": "Bernard",
        "lastName": "Swaniawski",
        "gender": "female",
        "pronoun": "she",
        "dateOfBirth": "1992-12-10",
        "language": {
          "code": "so",
          "name": ""
        },
        "nationality": {
          "code": "UZ",
          "name": ""
        },
        "jobTitle": "Assistant",
        "parentCompany": "Aquicore",
        "loyaltyPrograms": [],
        "identityProofs": null
      },
      {
        "modelVersion": "1.0",
        "id": "3168451881",
        "lastUpdatedOn": "2021-07-28T02:57:53.280621025Z",
        "createdOn": "2021-04-28T02:57:53.280621025Z",
        "lastUpdatedBy": "Jerad Homenick",
        "createdBy": "Ike Littel",
        "emails": [],
        "phones": [
          {
            "type": "",
            "number": "1-443-248-5876",
            "primary": false,
            "countryCode": 87
          },
          {
            "type": "home",
            "number": "539.557.5533",
            "primary": false,
            "countryCode": 36
          }
        ],
        "addresses": [
          {
            "type": "",
            "line1": "7809 Estates chester",
            "line2": "more content line 2",
            "line3": "more content line 3",
            "line4": "more content line 4",
            "city": "Memphis",
            "stateProvince": {
              "code": "ME",
              "name": ""
            },
            "postalCode": "82665",
            "country": {
              "code": "CN",
              "name": ""
            },
            "primary": false
          },
          {
            "type": "mailing",
            "line1": "704 East Skyway view",
            "line2": "more content line 2",
            "line3": "more content line 3",
            "line4": "more content line 4",
            "city": "Fremont",
            "stateProvince": {
              "code": "NM",
              "name": ""
            },
            "postalCode": "82999",
            "country": {
              "code": "GT",
              "name": ""
            },
            "primary": true
          }
        ],
        "honorific": "Mx",
        "firstName": "Fleta",
        "middleName": "Uriel",
        "lastName": "Balistreri",
        "gender": "other",
        "pronoun": "he",
        "dateOfBirth": "1913-02-05",
        "language": {
          "code": "or",
          "name": ""
        },
        "nationality": {
          "code": "CL",
          "name": ""
        },
        "jobTitle": "Executive",
        "parentCompany": "T. Rowe Price",
        "loyaltyPrograms": [],
        "identityProofs": null
      }
    ]
  },
  "paymentInformation": {
    "paymentType": "loyalty_redemption",
    "ccInfo": {
      "token": "",
      "cardType": "",
      "cardExp": "",
      "cardCvv": "",
      "expiration": "",
      "name": "",
      "address": {
        "type": "",
        "line1": "",
        "line2": "",
        "line3": "",
        "line4": "",
        "city": "",
        "stateProvince": {
          "code": "",
          "name": ""
        },
        "postalCode": "",
        "country": {
          "code": "",
          "name": ""
        },
        "primary": false
      }
    },
    "routingNumber": "",
    "accountNumber": "",
    "voucherID": "",
    "address": {
      "type": "",
      "line1": "56589 Trafficway shire",
      "line2": "more content line 2",
      "line3": "more content line 3",
      "line4": "more content line 4",
      "city": "New York City",
      "stateProvince": {
        "code": "TX",
        "name": ""
      },
      "postalCode": "34610",
      "country": {
        "code": "JM",
        "name": ""
      },
      "primary": true
    }
  },
  "promocode": "XXLYVK6ONS",
  "email": "",
  "phone": "",
  "status": "confirmed"
}