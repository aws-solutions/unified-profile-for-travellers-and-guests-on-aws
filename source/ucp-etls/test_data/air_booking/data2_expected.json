{"air_booking_recs": [{"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "booking_id": "TLIUNY", "segment_id": "TLIUNY-PEK-DEN", "from": "PEK", "to": "DEN", "flight_number": "RK0414", "departure_date": "2022-02-24", "departure_time": "02:48", "arrival_date": "2022-02-24", "arrival_time": "12:46", "channel": "mobile", "status": "confirmed", "honorific": "Ms", "first_name": "Krista", "middle_name": "Tony", "last_name": "Roob", "gender": "female", "pronoun": "she", "date_of_birth": "2022-05-26", "job_title": "Supervisor", "company": "CareSet Systems", "price": 0.0, "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "dangelowehner@stiedemann.info", "email_type": "business", "phone": "712.877.8663", "phone_type": "", "traveller_id": "1150274836", "address_type": "", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "booking_id": "TLIUNY", "segment_id": "TLIUNY-PEK-DEN", "from": "PEK", "to": "DEN", "flight_number": "RK0414", "departure_date": "2022-02-24", "departure_time": "02:48", "arrival_date": "2022-02-24", "arrival_time": "12:46", "channel": "mobile", "status": "confirmed", "honorific": "Ms", "first_name": "Tanya", "middle_name": "Constantin", "last_name": "Schmidt", "gender": "other", "pronoun": "they", "date_of_birth": "1986-05-10", "job_title": "Architect", "company": "Wheaton World Wide Moving", "price": 0.0, "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "6671697500", "address_type": "", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "booking_id": "TLIUNY", "segment_id": "TLIUNY-PEK-DEN", "from": "PEK", "to": "DEN", "flight_number": "RK0414", "departure_date": "2022-02-24", "departure_time": "02:48", "arrival_date": "2022-02-24", "arrival_time": "12:46", "channel": "mobile", "status": "confirmed", "honorific": "Ms", "first_name": "Cielo", "middle_name": "Bernard", "last_name": "Swaniawski", "gender": "female", "pronoun": "she", "date_of_birth": "1992-12-10", "job_title": "Assistant", "company": "Aquicore", "price": 0.0, "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email": "ricokemmer@powlowski.name", "email_type": "personal", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "traveller_id": "6759135163", "address_type": "", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": ""}, {"model_version": "1.0", "object_type": "air_booking", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "booking_id": "TLIUNY", "segment_id": "TLIUNY-PEK-DEN", "from": "PEK", "to": "DEN", "flight_number": "RK0414", "departure_date": "2022-02-24", "departure_time": "02:48", "arrival_date": "2022-02-24", "arrival_time": "12:46", "channel": "mobile", "status": "confirmed", "honorific": "Ms", "first_name": "Fleta", "middle_name": "Uriel", "last_name": "Balistreri", "gender": "other", "pronoun": "he", "date_of_birth": "1913-02-05", "job_title": "Executive", "company": "T. Rowe Price", "price": 0.0, "payment_type": "loyalty_redemption", "cc_token": "", "cc_type": "", "cc_exp": "", "cc_cvv": "", "cc_name": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_postal_codes": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_country": "", "email_business": "", "email": "", "phone": "1-443-248-5876", "phone_type": "", "traveller_id": "3168451881", "address_mailing_line1": "704 East Skyway view", "address_mailing_line2": "more content line 2", "address_mailing_line3": "more content line 3", "address_mailing_line4": "more content line 4", "address_mailing_postal_codes": "82999", "address_mailing_city": "Fremont", "address_mailing_state_province": "NM", "address_mailing_country": "GT", "address_type": "mailing"}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "address": "dangelowehner@stiedemann.info", "type": "business", "traveller_id": "1150274836"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "address": "ricokemmer@powlowski.name", "type": "personal", "traveller_id": "6759135163"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "address": "elisabethrunolfsson@kihn.io", "type": "personal", "traveller_id": "6759135163"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "address": "aliyahklein@erdman.name", "type": "business", "traveller_id": "6759135163"}], "common_phone_recs": [{"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "number": "712.877.8663", "country_code": 4, "type": "", "traveller_id": "1150274836"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "number": "6672245434", "country_code": 29, "type": "personal", "traveller_id": "1150274836"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "number": "1-443-248-5876", "country_code": 87, "type": "", "traveller_id": "3168451881"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "number": "539.557.5533", "country_code": 36, "type": "home", "traveller_id": "3168451881"}], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2021-10-07T15:47:19.902359909Z", "last_updated_by": "Victor Jakubowski", "id": "8922230558", "program_name": "reward", "miles": "8922230558", "miles_to_next_level": 4057.0, "level": "platinium", "joined": "2022-12-13T17:48:56.394398113Z", "traveller_id": "6671697500"}]}